src/studio/src/designer/frontend/packages/schema-editor/package.json

```diff
@@ -28,36 +28,36 @@
     "@rollup/plugin-json": "~4.1.0",
     "@rollup/plugin-node-resolve": "~11.2.1",
     "@testing-library/jest-dom": "~5.14.1",
-    "@testing-library/react": "~11.2.7",
+    "@testing-library/react": "~12.0.0",
```

Comment:
Did you run `npm test` after this change?
Since it is a dev dependency it might break the local environment on a major update.
From what I've read of in the "Braking Changes", it seems to not break anything we are using though.

Reply:
I ran all tests a few times.